### PR TITLE
Update custom-detection-rules.md

### PR DESCRIPTION
**Pull request by msbemba**
Added statement for Alert tittle and detection name that they need to be unique as the rule does not get created if the name already exists.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7384

File: microsoft-365/security/defender/custom-detection-rules.md [hunk 1]
```diff
@@ -109,9 +109,9 @@ DeviceEvents
 
 With the query in the query editor, select **Create detection rule** and specify the following alert details:
 
-- **Detection name**—name of the detection rule
+- **Detection name**—name of the detection rule and this needs to be unique.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/custom-detection-rules.md]:
```suggestion
- **Detection name**—name of the detection rule; this name needs to be unique.
```

File: microsoft-365/security/defender/custom-detection-rules.md [hunk 2]
```diff
@@ -109,9 +109,9 @@ DeviceEvents
...
 - **Frequency**—interval for running the query and taking action. [See additional guidance below](#rule-frequency)
-- **Alert title**—title displayed with alerts triggered by the rule
+- **Alert title**—title displayed with alerts triggered by the rule and this needs to be unique.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender/custom-detection-rules.md]:
```suggestion
- **Alert title**—title displayed with alerts triggered by the rule; this title needs to be unique.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@schmurky Please merge this PR. Thanks

**Comment by DulceMontemayor** [comment]
@yogkumgit, the base branch is out of date, please update and resend this merge request and i will approve, Thanks!

**Comment by yogkumgit** [comment]
@msbemba

**Comment by msbemba** [comment]
@yogkumgit Base branch has been updated.

**Comment by yogkumgit** [comment]
@DulceMontemayor Please merge. Thanks

**Comment by DulceMontemayor** [comment]
@yogkumgit please update the base branch, resubmit, and i will merge. Thank you!

**Pull request merged**